packages/teams-js/src/public/meeting.ts

```diff
@@ -414,4 +430,21 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.speakingStateChanged', handler);
   }
+
+  /**
+   * Registers a handler for changes to the selfParticipant raiseHand state. If the selfParticipant raises their hand, isHandRaised
+   * will be true. By default and if the selfParticipant hand is lowered, isHandRaised will be false. Only one handler can be registered
+   * at a time. A subsequent registration replaces an existing registration.
+   *
+   * @param handler The handler to invoke when the selfParticipant's raiseHand state changes.
```

Comment:
Would be good to add an @link to the documentation here that links back to IRaiseHandStateChangedEvent so that readers of the dev docs can easily find the type's documentation